application/src/test/java/org/opentripplanner/service/paging/PS2_ManyParetoOptimalItinerariesTest.java

```diff
@@ -134,11 +134,13 @@ void testPagingArriveByAndNext() {
 
     var nextCursor = subject.nextPageCursor();
 
+    // The earliest departure time is 12:26 here instead of 12:30,
+    // because it is determined from the latest itinerary departure from the results.
```

Comment:
```suggestion
    // The earliest departure time is 12:26 here, because it is determined from the 
    // latest itinerary departure from the results.
```

Reply:
I changed this comment